source/clear-linux/tutorials/web-server-install/web-server-install.rst

```diff
@@ -1,479 +1,424 @@
 
-Normally, root should only be allowed to connect from the 'localhost'.
-This ensures that someone cannot guess at the root password from the network.
+   .. code-block:: bash
 
-To block any remote root login, type 'y'.
+       ... Success!
+      Disallow root login remotely? [Y/n]
 
-.. code-block:: console
+   Normally, root should only be allowed to connect from the 'localhost'. This ensures that someone cannot guess the root password from the network. To block any remote root login, type 'y'.
 
-    ... Success!
-   Remove test database and access to it? [Y/n]
+   .. code-block:: bash
 
-By default, MariaDB comes with a database named 'test' which anyone can
-access. This database is also intended only for testing and should be
-removed.
+       ... Success!
+      Remove test database and access to it? [Y/n]
 
-To remove the test database, type 'y'.
+   By default, MariaDB includes a database named 'test' which anyone can access. This database is also intended only for testing and should be removed. To remove the test database, type 'y'.
 
-.. code-block:: console
+   .. code-block:: bash
 
-    - Dropping test database...
-    ... Success!
-    - Removing privileges on test database...
-    ... Success!
-   Reload privilege tables now? [Y/n]
+       - Dropping test database...
+       ... Success!
+       - Removing privileges on test database...
+       ... Success!
+      Reload privilege tables now? [Y/n]
 
-Reloading the privilege tables ensures all changes made so far take effect
-immediately.
+   Reloading the privilege tables ensures all changes made so far take effect immediately. To reload the privilege tables, type 'y'.
 
-To reload the privilege tables, type 'y'.
+   .. code-block:: bash
 
-.. code-block:: console
+       ... Success!
 
-    ... Success!
-
-   Cleaning up...
+      Cleaning up...
 
    All done!  If you've completed all of the above steps, your MariaDB
    installation should now be secure.
 
    Thanks for using MariaDB!
 
-The MariaDB installation is complete and we can now install phpMyAdmin to
-manage the databases.
+The MariaDB installation is complete and we can now install phpMyAdmin to manage the databases.
 
-Installing phpMyAdmin
+Install phpMyAdmin
 *********************
 
-The web-based tool phpMyAdmin is a straight-forward way to manage MySQL or
-MariaDB databases. Visit https://www.phpmyadmin.net for the complete
-discussion regarding phpMyAdmin, its documentation, the latest downloads, and
-other useful information.
+The web-based tool phpMyAdmin is a straightforward way to manage MySQL or MariaDB databases. Visit the `phpMyAdmin`_ website for the complete discussion regarding phpMyAdmin, its documentation, the latest downloads, and other useful information.
```

Comment:
This comment applies to Line 307; you addressed it in your push on May 23rd.
This URL needs to be formatted per the .rst convention. Please see our Contributor Guidelines: https://clearlinux.org/documentation/clear-linux/reference/collaboration/documentation/cross